src/mixins/input-popup/input-popup.ts

```diff
@@ -54,7 +54,7 @@ export class InputPopup extends ModulVue {
     }
 
     public hasValue(): boolean {
-        return this.internalValue !== undefined && this.internalValue !== '' && this.internalValue !== ' ';
+        return this.internalValue !== undefined && this.internalValue !== null && this.internalValue !== '' && this.internalValue !== ' ';
```

Comment:
return (this.internalValue || '').trim() !== '';

Reply:
Je pense que tu pourrais faire return !!(this.internalValue || '').trim() aussi ou un truc du genre

Reply:
C'est compact! 

J'ai appliqué ta suggestion, j'ai aussi mis un commentaire pour simplifier la compréhension de quelqu'un qui aurait à utiliser cette fonction.